pkg/clients/eks/fargateprofile_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	ekstypes "github.com/aws/aws-sdk-go-v2/service/eks/types"
 	"github.com/google/go-cmp/cmp"
 
-	"github.com/crossplane/provider-aws/apis/eks/v1alpha1"
+	v1alpha1 "github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"
```

Comment:
can we use here `manualv1alpha1` without `v1alpha1` and change the code like in identityproviderconfig

`"github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"`

Reply:
Fixed